src/docker/start.sh

```diff
@@ -10,9 +21,13 @@ sed -i "s/{postgres-user}/${POSTGRES_USER}/"              "$file"
 sed -i "s/{postgres-pass}/${POSTGRES_PASS}/"              "$file"
 
 file=${CATALINA_HOME}/conf/${PROJECT}_common_config.xml
-sed -i "s~{proxy-url}~${HTTP_PROXY}~"                     "$file"
-sed -i "s/{proxy-user}/${PROXY_USER}/"                    "$file"
-sed -i "s/{proxy-pass}/${PROXY_PASS}/"                    "$file"
+sed -i "s|{http-proxy-url}|${HTTP_PROXY_URL:-}|"              "$file"
+sed -i "s|{http-proxy-username}|${HTTP_PROXY_USERNAME:-}|"    "$file"
+sed -i "s|{http-proxy-password}|${HTTP_PROXY_PASSWORD:-}|"    "$file"
```

Comment:
The password could contain a `|` char. Do we like to error if it is the case?

Reply:
I would suggest accepting temporary this implementation and looking for a new solution in a second pull request, as described in https://github.com/samply/share-client/issues/60